src/Symfony/Component/Yaml/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Deprecate parsing duplicate mapping keys whose value is `null`
+ * Add support for dumping `null` as an empty string by using the `Yaml::DUMP_NULL_AS_EMPTY` flag
```

Comment:
I don’t think that „empty string“ is the correct term here. An empty string would be `''` or `""`.

Reply:
"As an empty value" maybe?